packages/legend-extension-application-studio-query-builder/src/components/MappingTestQueryBuilder.tsx

```diff
@@ -54,6 +54,11 @@ export const MappingTestQueryBuilder = observer(
         queryBuilderExtension.queryBuilderState.initialize(
           testState.queryState.query,
         );
+        await Promise.all([
```

Comment:
See my comment in `QueryBuilderChangeDetectionState`, we should only need to do:
`await queryBuilderState.changeDetectionState.start();` here